data_fetch.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+mkdir DATA
+osf -u 8kep2ppppt@2mailnext.com -p 5hju4 clone DATA
```

Comment:
So there is a nilearn fetcher for the MAIN dataset; it's just currently on Kamalaker's PR. See here, and the two subsequent functions: https://github.com/nilearn/nilearn/blob/16f2df26401e5b6a16ff134f4da71ff920e3ac40/nilearn/datasets/func.py#L1865
Do you think you can pip install that version of nilearn (just from the git commit; I can help you if you haven't done this before !) and use those functions instead of this OSF grab ?

Reply:
Ok, I fought the fetcher was for something else, ok I will use this version of nilean :+1: